### PR TITLE
1.8 release notes

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description
1.8 Release notes highlights:

- High Availability
- DevOps Pipelines
- User interface for Device Agent

## Tasks
- [x] Update YouTube Links
- [x] Add Highlight Image https://github.com/flowforge/website/issues/821
- [x] Add New Contributors
- [x] Add Documentation links
- [x] Add Bug fixes

### CODE REVIEW

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: Enhancing Node-RED with High Availability in FlowForge 1.8
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:2]:
```suggestion
title: FlowForge now offers High Availability Node-RED
```

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+subtitle: Empowering Node-RED with increased availability and streamlined deployment pipelines
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:3]:
```suggestion
subtitle: FlowForge 1.8 makes Node-RED applications more reliable and scalable, plus more streamline deployment pipelines.
```

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+date: 2023-06-08 
+authors: ["marian-demme"]
+#image: /blog/2023/05/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
+
+<!--more-->
+
+## High Availability
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:14]:
```suggestion
## More reliable and scalable Node-RED applications
```

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+Are you running business-critical processes in Node-RED and need to ensure they're always available and can handle increasing workloads? We heard you! We're excited to introduce High Availability (HA) in FlowForge to support your needs.
+
+Say you're managing a large influx of HTTP requests to your Node-RED instances. With our latest update, FlowForge now ensures that these requests are distributed evenly across your instances. This means your applications are less likely to experience downtime and can handle higher traffic loads​.
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:17]: Isn't the main point that we are creating two instances to run the same flow and a load balancer will distribute the load? I think you should explain this fact. Just sayin distributed evenly across your instances leaves a lot of unanswered questions, imho.

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+authors: ["marian-demme"]
+#image: /blog/2023/05/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
+
+<!--more-->
+
+## High Availability
+Are you running business-critical processes in Node-RED and need to ensure they're always available and can handle increasing workloads? We heard you! We're excited to introduce High Availability (HA) in FlowForge to support your needs.
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:15]: This section is awkward? If the answer to the first question is 'yes we are running business-critical process.....' then aren't they already being successful so they won't need the HA feature?  I would rework this section.

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+Say you're managing a large influx of HTTP requests to your Node-RED instances. With our latest update, FlowForge now ensures that these requests are distributed evenly across your instances. This means your applications are less likely to experience downtime and can handle higher traffic loads​.
+
+You might be wondering about editing flows in this new HA environment. For now, we've disabled editing in the HA environment to ensure consistency across instances. But don't worry, you can still develop your flows in a single instance and then seamlessly roll them out to your HA-ready production instance. See therefore also our next new feature, DevOps Pipelines. High Availability is our first [preview feature](https://flowforge.com/handbook/product/versioning/#preview-features), and your feedback is crucial. We encourage you to try out HA in your Node-RED instances and share your experiences with us. Your feedback will help us refine this feature and make it even better.
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:19]: Why is this important to be documenting in the release blog? This seems like it should be in documentation. Also the part about 'For now,' seems to indicate we will add this in the future. Is this true?

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+You can also get help on [the Node-RED forums](https://discourse.nodered.org/)
+
+As well as in the [forum within our Github project](https://github.com/flowforge/flowforge/discussions)
+
+Chat with us on the `#flowforge` channel on the [Node-RED Slack workspace](https://nodered.org/slack)
+
+You can raise a support ticket by emailing [support@flowforge.com](mailto:support@flowforge.com)
+
+We've also added a live chat widget to our website, you can access it using the icon on the bottom right corner of our website. We'd love to hear from you.
```
Comment (ZJvandeWeg) [inline src/blog/2023/06/flowforge-1-8-released.md:86]: Can we clean up this block?

I'd say we should link to the docs indeed, than just redirect everything else to community.flowforge.com?

```suggestion
## Getting help

Please check FlowForge's [documentation](https://flowforge.com/docs/) as the answers to many questions are covered there. Additionally you can go the the [community forum](https://community.flowforge.com) if you have
any feedback or feature requests.
```

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zS6P3RR86vE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## DevOps Pipelines
+
+Have you ever worried about the stability of new updates in your Node-RED production environment? Our new feature is designed to help you manage these concerns. DevOps Pipelines allow you to manage and track multiple deployments of your Node-RED instances, progressing through stages like test, development, staging, and production with ease​.
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:25]: We have already talked about devops pipelines. I think we need to emphasis what is new in 1.8?

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
+---
+title: FlowForge now offers High Availability Node-RED
+subtitle: FlowForge 1.8 makes Node-RED applications more reliable and scalable, plus more streamline deployment pipelines.
+description: FlowForge now supports High Availibility, DevOps Pipelines and more
+date: 2023-06-08 
+authors: ["marian-demme"]
+image: /blog/2023/06/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:10]: I think this section needs to be improved. I think it under sells the 1.8 features.  I would suggest something like the following:

FlowForge 1.8 introduces two key features that allow organizations to reliably deploy Node-RED applications into production. In 1.8, it is now possible to run Node-RED applications with high availability so the application is more scalable and more fault tolerant. FlowForge 1.8 also introduces software deliver pipelines so development teams can now setup dev/test/production environments for their Node-RED applications.

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+<!--more-->
+
+## More reliable and scalable Node-RED applications
+Do you want to run business-critical processes in Node-RED and need to ensure they are always available and can handle increasing workloads? We're excited to introduce High Availability (HA) in FlowForge to support your needs. Suppose you're managing a large influx of HTTP requests to your Node-RED instances. With our latest update, FlowForge now ensures that these requests are evenly distributed across two instances. This means your applications are less likely to experience downtime and can manage higher traffic loads. You can easily activate HA mode during the creation of an instance. For more details, please see our [documentation](https://flowforge.com/docs/user/).
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:15]: I think we want to talk about adding HA to Node-RED not FlowForge. We should also be more technical in describing what the feature does. I would also not start a paragraph with a sentence in this type of blog.  I would suggest something like the following:

FlowForge now makes it possible to deploy business critical applications built in Node-RED that are reliable and scalable. The new 1.8 features allows a Node-RED instance to be deployed in high availability mode, meaning two instances of the same Node-RED flows are available behind a load balancer. This allows for increased traffic to be automatically distributed across the two Node-RED instances. This means your Node-RED applications can handle more traffic and experience less downtime.

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+High Availability is our first [preview feature](https://flowforge.com/handbook/product/versioning/#preview-features), and your feedback is crucial. We encourage you to try out HA in your Node-RED instances and share your experiences with us. Your feedback will help us refine this feature and make it even better.
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:17]: We should be linking to documentation on how to try it out?

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zS6P3RR86vE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## DevOps Pipelines
+
+DevOps Pipelines enable you to manage and track multiple deployments of your Node-RED instances, effortlessly progressing through stages such as testing, development, staging, and production. In [FlowForge Version 1.4](../../02/flowforge-1-4-0-released),  we introduced the "Support for Staged Development". Now, we're taking it one step further. Within your applications section, you are now able to create your own DevOps Pipeline and track the progress across each instance in one view.
```
Comment (iskerrett) [inline src/blog/2023/06/flowforge-1-8-released.md:23]: I would suggest rewording as follows:

FlowForge 1.8 introduces the concept of Pipelines to better organize your Node-RED development. Development team can now setup different staging environments for different steps in the development cycle, ex. test, development and production. Node-RED instances can be pushed along a pipeline as they move along the development process. This allows for a better organized and predictable development process for your team.

The new Pipelines feature builds upon the Staged Development support that we introduced in 1.4.

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 11 above)
Comment (MarianRaphael) [reply src/blog/2023/06/flowforge-1-8-released.md:17]: Absolutely, it's just a placeholder for the tests. The documentation is not ready yet.

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 12 above)
Comment (MarianRaphael) [reply src/blog/2023/06/flowforge-1-8-released.md:23]:
```suggestion
FlowForge 1.8 introduces the concept of Pipelines to better organize your Node-RED development. Development team can now setup different staging environments for different steps in the development cycle, ex. test, development and production. Node-RED instances can be pushed along a pipeline as they move along the development process. This allows for a better organized and predictable development process for your team. 

The new Pipelines feature builds upon the Staged Development support that we introduced in[FlowForge Version 1.4](../../02/flowforge-1-4-0-released).
```

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 9 above)
Comment (ZJvandeWeg) [reply src/blog/2023/06/flowforge-1-8-released.md:10]: > It is now possible to run Node-RED applications with high availability so the application is more scalable and more fault tolerant.

Less comma's should be better

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 11 above)
Comment (ZJvandeWeg) [reply src/blog/2023/06/flowforge-1-8-released.md:17]: Please also offer them a 30 day trial license for their self-managed installs on Kubernetes by booking a demo at flowforge.com/book-demo

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 12 above)
Comment (ZJvandeWeg) [reply src/blog/2023/06/flowforge-1-8-released.md:23]: @MarianRaphael I think that's much better already, can we add a line that explains they no longer should develop their flows in production instances? While you implicitly touch on it, it might be good to make it explicit.

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+Imagine this: Your industrial equipment arrives with the Device Agent preinstalled. In the past, you might have faced challenges in configuring and connecting your device with FlowForge, particularly if you had no direct shell access. But not any more.
+
+With the newly introduced UI, you can easily set up and connect your device with FlowForge without needing to access the command line interface directly. This simplifies the process significantly and saves you time. For more details, see our [documentation](https://flowforge.com/docs/user/devices/).
+
+## Node-RED 3.1 Beta 3 Available
+
+FlowForge Cloud is a great place to try out the new Node-RED features, with FlowForge Cloud now including the [Node-RED 3.1.0-beta.3](https://discourse.nodered.org/t/node-red-3-1-0-beta-3-released/78716). If you want to try this version you can [duplicate your application](https://flowforge.com/docs/user/project-settings/#copy-project) or [upgrade your stack](https://flowforge.com/docs/user/changestack/).
```
Comment (ZJvandeWeg) [inline src/blog/2023/06/flowforge-1-8-released.md:37]: You're duplicating an instance, not an application. :) 

Further, shouldn't they 'fork' production to a development instance and than change the stack?

Comment (MarianRaphael) [reply src/blog/2023/06/flowforge-1-8-released.md]: Changed with: [change application to instance](https://github.com/flowforge/website/pull/820/commits/0ae7d825a01704571e5a14fa136ea04bd721f257)

Yes, I mean "duplication" is the terminology used in the FF settings. I replaced the "or" with "and". If that's what you mean

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 12 above)
Comment (MarianRaphael) [reply src/blog/2023/06/flowforge-1-8-released.md:23]:
```suggestion
FlowForge 1.8 introduces the concept of Pipelines to better organize your Node-RED development. Development team can now set up different staging environments for different steps in the development cycle, ex. test, development and production. Node-RED instances can be pushed along a pipeline as they move along the development process. This allows for a better organized and predictable development process for your team. 

The new Pipelines feature builds upon the Staged Development support that we introduced in[FlowForge Version 1.4](../../02/flowforge-1-4-0-released). We highly recommend that development teams avoid developing their flows directly in production instances. This approach fosters a more reliable and robust development process, reducing the risks associated with production environment modifications. Instead, start your development in a dedicated development or test instance and then promote your Node-RED instances to production once they have been thoroughly tested and reviewed. 
```

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 10 above)
Comment (MarianRaphael) [reply src/blog/2023/06/flowforge-1-8-released.md:15]:
```suggestion
FlowForge now makes it possible to deploy business critical applications built in Node-RED that are reliable and scalable. The new 1.8 features allows a Node-RED instance to be deployed in high availability mode, meaning two instances of the same Node-RED flows are available behind a load balancer. This allows for increased traffic to be automatically distributed across the two Node-RED instances. This means your Node-RED applications can handle more traffic and experience less downtime. For more details, please see our [documentation](https://flowforge.com/docs/user/high-availability).

Additionally, we're pleased to offer a 30-day premium trial license for self-managed installs on Kubernetes. To avail of this offer, book a demo at [flowforge.com/book-demo](https://flowforge.com/book-demo).
```

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 11 above)
Comment (MarianRaphael) [reply src/blog/2023/06/flowforge-1-8-released.md:17]: See: https://github.com/flowforge/website/pull/820#discussion_r1222579519

File: src/blog/2023/06/flowforge-1-8-released.md (hunk 9 above)
Comment (MarianRaphael) [reply src/blog/2023/06/flowforge-1-8-released.md:10]:
```suggestion
FlowForge 1.8 introduces two key features that allow organizations to reliably deploy Node-RED applications into production. In 1.8, it is now possible to run Node-RED applications with high availability so the application is more scalable and more fault tolerant. FlowForge 1.8 also introduces software deliver pipelines, so development teams can now set up dev/test/production environments for their Node-RED applications.
```

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+
+The new Pipelines feature builds upon the Staged Development support that we introduced in[FlowForge Version 1.4](../../02/flowforge-1-4-0-released). We highly recommend that development teams avoid developing their flows directly in production instances. This approach fosters a more reliable and robust development process, reducing the risks associated with production environment modifications. Instead, start your development in a dedicated development or test instance and then deploy your Node-RED instance to production once they have been thoroughly tested and reviewed. 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/S--viuPhrS8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (robmarcer) [inline src/blog/2023/06/flowforge-1-8-released.md:29]:
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/Pbql22f3vqY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: src/blog/2023/06/flowforge-1-8-released.md [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+
+Additionally, we're pleased to offer a 30-day premium trial license for self-managed installs on Kubernetes. To avail of this offer, book a demo at [flowforge.com/book-demo](https://flowforge.com/book-demo).
+
+High Availability is our first [preview feature](https://flowforge.com/handbook/product/versioning/#preview-features), and your feedback is crucial. We encourage you to try out HA in your Node-RED instances and share your experiences with us. Your feedback will help us refine this feature and make it even better.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zS6P3RR86vE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (robmarcer) [inline src/blog/2023/06/flowforge-1-8-released.md:21]:
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/mbDkjKhVwIw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Review by robmarcer — CHANGES_REQUESTED** [review]

**Review by ZJvandeWeg — APPROVED** [review]

**Review by iskerrett — APPROVED** [review]
